test/Windows.hs

```diff
@@ -58,35 +59,51 @@ restrictions =
 
 -- | The 'dirname' operation.
 operationDirname :: Spec
-operationDirname = do
-  it
-    "dirname ($(mkAbsDir parent) </> $(mkRelFile dirname)) == dirname $(mkRelFile dirname) (unit test)"
-    (dirname ($(mkAbsDir "C:\\chris\\") </> $(mkRelDir "bar")) ==
-     dirname $(mkRelDir "bar"))
-  it
-    "dirname ($(mkRelDir parent) </> $(mkRelFile dirname)) == dirname $(mkRelFile dirname) (unit test)"
-    (dirname ($(mkRelDir "home\\chris\\") </> $(mkRelDir "bar")) ==
-     dirname $(mkRelDir "bar"))
-  it
-    "dirname $(mkRelDir .) == $(mkRelDir .)"
-    (dirname $(mkRelDir ".") == $(mkRelDir "."))
-  it
-    "dirname C:\\ must be a Rel path"
-    ((parseAbsDir $ show $ dirname (fromJust (parseAbsDir "C:\\"))
-     :: Maybe (Path Abs Dir)) == Nothing)
+operationDirname =
+  do it "dirname ($(mkAbsDir parent) </> $(mkRelFile dirname)) == dirname $(mkRelFile dirname) (absolute)"
+        (dirnamesShouldBeEqual
+          ($(mkAbsDir "C:\\chris\\") </> $(mkRelDir "bar"))
+           $(mkRelDir "bar"))
```

Comment:
To me, the additional indentation of the second argument to `dirnamesShouldBeEqual` suggests that it is inside the parentheses opened in the line above. But it's not an important issue, you don't have to change it.